moztelemetry/store.py

```diff
@@ -36,7 +38,12 @@ def get_key(self, key):
         try:
             # get_key must return a file-like object because that's what's
             # required by parse_heka_message
-            return bucket.Object(key).get()['Body']
+            s3object = bucket.Object(key).get()
+            if s3object['ResponseMetadata']['HTTPHeaders'].get(
```

Comment:
According to the [boto3 docs](https://boto3.readthedocs.io/en/latest/reference/services/s3.html#S3.Object.get) there should be a `ContentEncoding` key in the returned dictionary